lib/core/src/main/java/io/atlasmap/actions/StringComplexFieldActions.java

```diff
@@ -1,362 +0,0 @@
-/**
- * Copyright (C) 2017 Red Hat, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *         http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.atlasmap.actions;
-
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Locale;
-import java.util.Map;
-import java.util.UUID;
-import java.util.regex.Pattern;
-
-import io.atlasmap.api.AtlasFieldAction;
-import io.atlasmap.spi.AtlasFieldActionInfo;
-import io.atlasmap.v2.Action;
-import io.atlasmap.v2.Append;
-import io.atlasmap.v2.CollectionType;
-import io.atlasmap.v2.Concatenate;
-import io.atlasmap.v2.EndsWith;
-import io.atlasmap.v2.FieldType;
-import io.atlasmap.v2.Format;
-import io.atlasmap.v2.IndexOf;
-import io.atlasmap.v2.LastIndexOf;
-import io.atlasmap.v2.PadStringLeft;
-import io.atlasmap.v2.PadStringRight;
-import io.atlasmap.v2.Prepend;
-import io.atlasmap.v2.ReplaceAll;
-import io.atlasmap.v2.ReplaceFirst;
-import io.atlasmap.v2.Split;
-import io.atlasmap.v2.StartsWith;
-import io.atlasmap.v2.SubString;
-import io.atlasmap.v2.SubStringAfter;
-import io.atlasmap.v2.SubStringBefore;
-
-public class StringComplexFieldActions implements AtlasFieldAction {
-
-    public static final String STRING_SEPARATOR_REGEX = "^\\s+:_+=";
-    public static final Pattern STRING_SEPARATOR_PATTERN = Pattern.compile(STRING_SEPARATOR_REGEX);
-
-    @AtlasFieldActionInfo(name = "Append", sourceType = FieldType.STRING, targetType = FieldType.STRING, sourceCollectionType = CollectionType.NONE, targetCollectionType = CollectionType.NONE)
-    public static String append(Action action, Object input) {
```

Comment:
This is where the Append action's execution method exists before this PR, amongst all the other action methods that are related to strings and that have parameters.